static/app/views/onboarding/setupDocs.tsx

```diff
@@ -68,81 +68,163 @@ function MissingExampleWarning({
   );
 }
 
+function LoadGettingStartedDoc({
+  platform,
+  orgSlug,
+  projectSlug,
+  activeProductSelection,
+}: {
+  activeProductSelection: PRODUCT[];
+  orgSlug: Organization['slug'];
+  platform: PlatformIntegration | null;
+  projectSlug: Project['slug'];
+}) {
+  const [module, setModule] = useState<null | {
+    default: React.ComponentType<{activeProductSelection: PRODUCT[]; dsn: string}>;
+  }>(null);
+
+  const platformPath =
+    platform?.type === 'framework'
+      ? platform?.id.replace(`${platform.language}-`, `${platform.language}/`)
+      : platform?.id;
+
+  const {
+    data: projectKeys = [],
+    isError: projectKeysIsError,
+    isLoading: projectKeysIsLoading,
+  } = useApiQuery<ProjectKey[]>([`/projects/${orgSlug}/${projectSlug}/keys/`], {
+    staleTime: Infinity,
+  });
+
+  useEffect(() => {
+    if (projectKeysIsError || projectKeysIsLoading) {
+      return;
+    }
+
+    async function getGettingStartedDoc() {
+      const mod = await import(
+        /* webpackExclude: /.spec/ */
+        `sentry/gettingStartedDocs/${platformPath}`
+      );
+      setModule(mod);
+    }
+    getGettingStartedDoc();
+  }, [platformPath, projectKeysIsError, projectKeysIsLoading, projectKeys]);
+
+  if (!module) {
+    return null;
+  }
+
+  const {default: GettingStartedDoc} = module;
+  return (
+    <GettingStartedDoc
+      dsn={projectKeys[0].dsn.public}
```

Comment:
might want to check projectKeys length

Reply:
in the onboarding flow the `projectKeys` shall be theoretically always of length 1, but in case the user creates more keys and go to the instructions after onboarding, which one are we displaying atm? 🤔